pkg/crc/api/helpers.go

```diff
@@ -40,6 +40,12 @@ func (c *context) String(code int, r string) error {
 	return err
 }
 
+func (c *context) Code(code int) error {
+	var err error
+	c.code = code
+	return err
```

Comment:
could be just `return nil`?

Reply:
yeah, i just followed the same pattern as the other methods